src/main/java/teachmyskill/task/database/Connect.java

```diff
@@ -0,0 +1,31 @@
+package teachmyskill.task.database;
+
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import static java.sql.DriverManager.getConnection;
+
+public class Connect {
+
+    public static Connection getConnect() {
+
+        Driver driver = new org.postgresql.Driver();

```

Comment:
In this implementation, it seems, you will register driver every time you call this method. It's not correct approach. Remember my example from the lesson